core-features/generate-client-and-cli-install-pnpm-lockfile/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eux
+
+VERSION=$(cat ../../.github/prisma-version.txt)
+
+cp  -r ../_common/generate-client-and-cli-install/* .
+
+# global instead instead
+npm install -g prisma@$VERSION
```

Comment:
why global?

Reply:
Improved with "install prisma globally to test that it can install the client & cli"